components/remote_settings/src/remote_settings.udl

```diff
@@ -8,9 +8,9 @@ typedef string RsJsonObject;
 namespace remote_settings {};
 
 dictionary RemoteSettingsConfig {
-    string? server_url = null;
-    string? bucket_name = null;
     string collection_name;
+    string? bucket_name = null;
+    string? server_url = null;
```

Comment:
Nit: the URL is more significant than the `bucket_name` or `collection_name`; would you consider having the url before the bucket?

Reply:
It is, but its also probably the most likely to be defaulted vs the other two. I think either this order or the previous order with no default value support are the most correct options. I'm going to keep this as is.